packages/contracts/contracts/L1/rollup/StateCommitmentChain.sol

```diff
@@ -294,15 +319,44 @@ contract StateCommitmentChain is IStateCommitmentChain, Lib_AddressResolver {
         emit StateBatchDeleted(_batchHeader.batchIndex, _batchHeader.batchRoot);
     }
 
+    /**
+     * Checks that a batch header matches the stored hash for the given index.
+     * @param _batch Submit batch data.
+     * @param _shouldStartAtElement or not the header matches the stored one.
+     */
+    function _distributeTssReward(bytes32[] memory _batch, uint256 _shouldStartAtElement) internal {
+        // get address of tss group member
+        address[] memory tssMembers = ITssGroupManager(resolve("TssGroupManager")).getTssGroupUnJailMembers();
+        require(tssMembers.length > 0, "get tss members in error");
+
+        // construct calldata for claimReward call
+        bytes memory message = abi.encodeWithSelector(
+            ITssRewardContract.claimReward.selector,
+            _shouldStartAtElement,
```

Comment:
is it equal to L2 start block number?

Reply:
yes

Reply:
all the shift height used between L1 and L2 contract is the L2 block height